src/sources/journald.rs

```diff
@@ -75,8 +75,9 @@ pub struct JournaldConfig {
     pub data_dir: Option<PathBuf>,
     pub batch_size: Option<usize>,
     pub journalctl_path: Option<PathBuf>,
+    /// Deprecated
     #[serde(default)]
-    pub remap_priority: bool,
+    remap_priority: bool,
```

Comment:
Does this need to go back to `pub`?

Reply:
Only if it's needed internally for tests.